### PR TITLE
Preserve current tab across prev/next navigation

### PR DESCRIPTION
**Pull request by distantnative**
### Enhancement
- Panel: current tab is preserved when navigation to previous/next model
https://kirby.nolt.io/412

### Breaking changes
None

## Ready?

- [x] Unit tests for fixed bug/feature
- [x] In-code documentation (wherever needed)
- [x] Tests and checks all pass

### For review team

- [x] Add changes to release notes draft in Notion
- [x] Add to [website docs release checklist](https://github.com/getkirby/getkirby.com/pulls) (if needed)

### CODE REVIEW

**Comment by distantnative** [comment]
@afbora would you have time to test this with me?

**Comment by afbora** [comment]
I like that! Works great for me on single-language, multi-language and between different blueprints setup 👍

File: src/Panel/Model.php [hunk 1]
```diff
@@ -387,6 +387,32 @@ public function toLink(string $tooltip = 'title'): array
...
+    protected function toPrevNextLink($model = null, string $tooltip = 'title'): ?array
+    {
+        if ($model === null) {
+            return null;
+        }
+
+        $data = $model->panel()->toLink($tooltip);
+
+        if ($tab = get('tab')) {
+            $data['link'] .= '?tab=' . $tab;
```
Comment (afbora) [inline src/Panel/Model.php]: This is more Kirby style 🙂 

```suggestion
            $uri = new Uri($data['link'], [
                'query' => ['tab' => $tab]
            ]);

            $data['link'] = $uri->toString();
```

**Comment by distantnative** [comment]
Great, so only need to figure out some unit tests.

**Comment by afbora** [comment]
Anything I can help with?

**Comment by distantnative** [comment]
If you want to give it a try, please! :)

**Comment by afbora** [comment]
@distantnative I've done but sorry for mixing commits. Could you squash the commits please? 😇 

Edit: I've fixed the commits ✅

**Review by afbora — APPROVED** [review]

**Review by bastianallgeier — APPROVED** [review]

**Pull request merged**